addons/account_peppol/exceptions.py

```diff
@@ -0,0 +1,100 @@
+    22: lambda: _('An intermediary MSH has assigned the message to an MPC for pulling and has successfully stored it. However the intermediary set a limit on the time it was prepared to wait for the message to be pulled, and that limit has been reached.'),
+    23: lambda: _('An MSH has determined that the message is expired and will not attempt to forward or deliver it.'),
+    30: lambda: _('The structure of a received bundle is not in accordance with the bundling rules.'),
+    31: lambda: _('A message unit in a bundle was not processed because a related message unit in the bundle caused an error.'),
+    40: lambda: _('A fragment is received that relates to a group that was previously rejected.'),
+    41: lambda: _('A fragment is received but more than one fragment message in a group of fragments specifies a value for this element.'),
+    42: lambda: _('A fragment is received but more than one fragment message in a group of fragments specifies a value for this element.'),
+    43: lambda: _('A fragment is received but more than one fragment message in a group of fragments specifies a value for this element.'),
+    44: lambda: _('A fragment is received but more than one fragment message in a group of fragments specifies a value for this element.'),
+    45: lambda: _('A fragment is received but more than one fragment message in a group of fragments specifies a value for a compression element.'),
+    46: lambda: _('A fragment is received but a previously received fragment message had the same values for GroupId and FragmentNum'),
+    47: lambda: _('The href attribute does not reference a valid MIME data part, MIME parts other than the fragment header and a data part are in the message. are added or the SOAP Body is not empty.'),
+    48: lambda: _('An incoming message fragment has a a value greater than the known FragmentCount.'),
+    49: lambda: _('A value is set for FragmentCount, but a previously received fragment had a greature value.'),
+    50: lambda: _('The size of the data part in a fragment message is greater than %s'),
+    51: lambda: _('More time than %s has passed since the first fragment was received but not all other fragments are received.'),
+    52: lambda: _('Message properties were present in the fragment SOAP header that were not specified in %s'),
+    53: lambda: _('The eb3:Message header copied to the fragment header does not match the eb3:Message header in the reassembled source message.'),
+    54: lambda: _('Not enough disk space available to store all (expected) fragments of the group.'),
+    55: lambda: _('An error occurred while decompressing the reassembled message.'),
+    60: lambda: _('A responding MSH indicates that it applies the alternate MEP binding to the response message.'),
+}
+
+def get_exception_message(peppol_code, default_message=""):
+    if peppol_code not in EXCEPTION_CODE_MESSAGES_MAP:
+        return default_message
+    return EXCEPTION_CODE_MESSAGES_MAP[peppol_code]()
```

Comment:
```suggestion
def get_exception_message(error_vals):
    peppol_code = error_vals['code']
    return EXCEPTION_CODE_MESSAGES_MAP.get(peppol_code, _("Unknown Peppol Error: %s", error_vals))
```
Maybe that's just easier to give the dict of `error_vals` ? You're anyway already passing them somehow through the default_message. 
Retrieve the code in this method and no need to pass a default error that is always the same ?
Same below.